src/console_cmds.cpp

```diff
@@ -53,6 +53,22 @@
 /* scriptfile handling */
 static uint _script_current_depth; ///< Depth of scripts running (used to abort execution when #ConReturn is encountered).
 
+/** Scheduled execution handling */
+static std::string _scheduled_script; ///< Script scheduled to execute by the 'schedule' console command (empty if no script is scheduled)
+static IntervalTimer<TimerGameCalendar> _schedule_timer = {{TimerGameCalendar::MONTH, TimerGameCalendar::Priority::NONE}, [](auto) {
+	if (_scheduled_script.empty()) {
+		return;
+	}
+
+	/** Clear the schedule before rather than after the script to allow the script to itself call
```

Comment:
Just `/* xxx */` for regular comments, `/** xxx */` is for Doxygen.

Reply:
Oh!

Reply:
Like now?

Reply:
(I'm unsure about the top level `Scheduled execution handling`. I see https://github.com/OpenTTD/OpenTTD/blob/d7eb29d2922e10cd1f2fa8d7bba8069931e0a3f6/src/company_gui.cpp#L55 seems to be something fairly similar that does use the Doxygen format)

Reply:
Yeah, the top level one shouldn't be Doxygen-style either.

Reply:
Like now?